apps/storefront/src/index.html

```diff
@@ -6,5 +6,72 @@
   <body>
     <oryx-app></oryx-app>
   </body>
+  <script>
+    const EVENTS_DATA = Symbol.for('EVENTS_DATA');
+    const replayableAttribute = 'replayable';
```

Comment:
gpt suggests: 

```ts
const EVENTS_DATA = Symbol.for('EVENTS_DATA');
const replayableAttribute = 'replayable';

// Function to add events to elements with the replayable attribute
const addEventsAction = () => {
  const elements = document.querySelectorAll(`[${replayableAttribute}]`);

  elements.forEach(element => {
    element[EVENTS_DATA] = {
      events: [],
      listener: (event) => {
        event.stopPropagation();
        event.preventDefault();
        element[EVENTS_DATA]?.events.push(event);
      },
    };

    const events = element.getAttribute(replayableAttribute).split(':');

    events.forEach(eventType => {
      element.addEventListener(eventType, element[EVENTS_DATA].listener);
    });
  });
};

// Check if shadow DOM exists in oryx-app
const oryxApp = document.querySelector('oryx-app');
if (oryxApp && oryxApp.shadowRoot) {
  addEventsAction();
}

```

Reply:
Why we dropped treewalk here and also use forEach instead of `for of` which is slower?

Reply:
I wasn't aware of the performance decrease, also no opinions on the code so much. Was more coming from smaller code lines ;)